exercises/pig-latin/src/example.c

```diff
@@ -20,8 +20,8 @@ static bool starts_with_vowel_sound(const char *input)
        || (first_letter == 'y' && sec_letter == 't');
 }
 
-static void count_words_and_chars(const char *input, size_t * words,
-                                  size_t * chars)
+static void count_words_and_chars(const char *input, size_t *words,
+                                  size_t *chars)
```

Comment:
```suggestion
static void count_words_and_chars(const char *input, size_t * words,
                                  size_t * chars)
```

OMG @ryanplusplus this spacing by indent on the pointers is cray cray. What is the logic on the first parameter not having a space after the asterisk but the rest of them doing so?